src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/EntityAIWorkCowboy.java

```diff
@@ -93,7 +92,7 @@ private AIState milkCows()
     {
         worker.setLatestStatus(new TextComponentTranslation(TranslationConstants.COM_MINECOLONIES_COREMOD_STATUS_COWBOY_MILKING));
 
-        if (!worker.hasItemInInventory(getBreedingItem(), 0) && isInHut(new ItemStack(Items.BUCKET, 1)))
+        if (!worker.hasItemInInventory(getBreedingItem().getItem(), 0) && isInHut(new ItemStack(Items.BUCKET, 1)))
```

Comment:
Use inventoryutils for checks like this since that is itemstack agnostic, it does not matter here all that much since you compare wheat which has no meta or nbt. But Keep it in mind for the future.